x/ccv/provider/keeper/genesis.go

```diff
@@ -140,5 +188,6 @@ func (k Keeper) ExportGenesis(ctx sdk.Context) *types.GenesisState {
 		k.GetAllValidatorConsumerPubKeys(ctx, nil),
 		k.GetAllValidatorsByConsumerAddr(ctx, nil),
 		consumerAddrsToPrune,
+		lastProviderConsensusValSet,
```

Comment:
Why do we export this if we never use it on import?

Reply:
Right, it doesn't need to be part of the genesis state actually. Changed